guide/src/examples/wasm-in-web-worker.md

```diff
@@ -15,6 +16,15 @@ Firefox does not. To have compatibility across browsers, the whole example is
 set up without relying on ES modules as target. Therefore we have to build
 with `--target no-modules`. The full command can be found in `build.sh`.
 
+However note that this may no longer be true when you read this. The
+[related bug][ff_worker_module_bug] has been closed. ES modules in web workers
+already work in Firefox 111.a nightly build. If you want to check if ES
+modules work as target for web workers in your current browser, check out if
+[this example][wasm_worker_with_modules] behaves as expected.
+
+[ff_worker_module_bug]: https://bugzilla.mozilla.org/show_bug.cgi?id=1247687
+[wasm_worker_with_modules]: https://sgasse.github.io/wasm_worker_interaction/wasm_module_js_worker/www/index.html
```

Comment:
I don't think we need this information here. But we can update the note above with Firefox 114, which is the version expected to ship worker modules right now.